src/chapters/vstools/studio.rst

```diff
@@ -0,0 +1,90 @@
+Studio: If It Ain't Broke, Add a Breakpoint!
+============================================
+
+Introduction 
+--------------
+
+The purpose of this studio is to talk about your current debugging strategies and how to make the most of the debugger tools discussed in this chapter.
+
+Before we start practicing with debugging tools, go over with the group one error you encountered with any exercise, studio, or book example using Visual Studio.
+This could be the result of a typo or a logical error. 
+
+#. What was the error?
+#. How did you solve this error? What have been the strategies and tools you have been using so far to debug your code?
+#. Could one of the debugging tools help you when addressing this error?
+   For example, if you encountered an error where scope was interferring with access to data, could you find the accessibility of the data with a debugging tool?
+
+Part 1: Calculate The Area Of A Circle
+-----------------------------------------
+
+As you build your code for today's studio, explore the colorations, icons, and other intellisense features of Visual Studio.
+Don't forget to try the debugger if something isn't working as you planned, or even if it is working as you planned.
+
+
+**To get started:** Create a new console application in Visual Studio for the studio.
```

Comment:
might not be a bad idea to link back to the "create a new project" section in visual studio setup. this is one of the first times we're asking students to create a VS project from scratch